migrations/dev-20210818-add-balance-transfer-transaction-kind.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+module.exports = {
+  up: async queryInterface => {
+    await queryInterface.sequelize.query(`
+      ALTER TYPE "enum_Transactions_kind"
+      ADD VALUE IF NOT EXISTS 'BALANCE_TRANSFER' AFTER 'PREPAID_PAYMENT_METHOD'
```

Comment:
Why not this, to keep it in alphabetical order?

```suggestion
      ADD VALUE IF NOT EXISTS 'BALANCE_TRANSFER' AFTER 'ADDED_FUNDS'
```

Reply:
Ops, yes that makes sense. Done. 👍🏽